app/models/role.rb

```diff
@@ -0,0 +1,14 @@
+class Role < ApplicationRecord
+  has_and_belongs_to_many :users, :join_table => :users_roles
+  
+  belongs_to :resource,
+             :polymorphic => true,
```

Comment:
:polymorphic => true is the default setting when the gem is installed. My understanding is that this allows roles to be assigned to different types of objects. While I don't think this would be used in the :admin role, removing this line is a global setting. So in the future if there was a scenario where we needed a role applied to multiple objects we would have to re-add. Could remove or leave in this scenario

Reply:
For the sake of keeping things moving lets just keep it in as it causes no harm.